regression-pce.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
```

Comment:
Do we need to add this?

This https://github.com/Dasharo/open-source-firmware-validation/tree/develop?tab=readme-ov-file#running-regression-tests would work here?

Reply:
Right, the file can be dropped.

Reply:
Removed in https://github.com/Dasharo/open-source-firmware-validation/pull/237/commits/ea98c9876628026d4e03cd8cfdc45694870ad1cd